lib/vmdb/loggers.rb

```diff
@@ -50,20 +53,34 @@ def self.create_logger(log_file_name, logger_class = VMDBLogger)
       configure_external_loggers
     end
 
-    private_class_method def self.create_container_logger
-      return unless ENV["CONTAINER"]
+    private_class_method def self.create_broadcast_logger
+      return create_container_logger if container_logger?
+      return create_journald_logger  if journald_logger?
+
+      nil
+    end
 
+    private_class_method def self.container_logger?
+      !!ENV["CONTAINER"]
+    end
+
+    private_class_method def self.create_container_logger
       require "manageiq/loggers/container"
       ManageIQ::Loggers::Container.new
     end
 
-    private_class_method def self.create_journald_logger
-      return unless MiqEnvironment::Command.supports_systemd?
+    private_class_method def self.journald_logger?
+      return false unless MiqEnvironment::Command.supports_systemd?
 
       require "manageiq/loggers/journald"
-      ManageIQ::Loggers::Journald.new
+      true
```

Comment:
again. I understand the `true` here, but why the `false`s?

Are we going towards a more explicit style? (This is a change that I want to follow if that is your intent. Just want to make it explicit so the whole team knows)

Reply:
I felt the check method read cleaner as a separate method than the create method - originally I was also going to use that method in the specs, but ended up not doing it.

I could see just using the create methods directly though - might be cleaner in the end to just do 

```ruby
def create_broadcast_logger
  create_container_logger || create_journald_logger
end
```

Reply:
Ok I went with your approach - updated.